src/style.css

```diff
@@ -61,16 +65,16 @@ nav {
 
   /* background-position: right bottom; */
```

Comment:
- [ ] 🙏Please remove the commented code from the project. This will help make your code cleaner😉.